_1327/settings.py

```diff
@@ -33,6 +33,9 @@
 ENABLE_DEBUG_TOOLBAR = False
 ENABLE_DEBUG_TOOLBAR = ENABLE_DEBUG_TOOLBAR and DEBUG and not TESTING
 
+# wip, disabled by default
```

Comment:
should that remain wip?

Reply:
As I continuously fail to exclude the tenca part it from the CI-runner or simply send a user-message instead of throwing exceptions---yes.

Will investigate in the next two days and remove the note when I'm done.

Reply:
removed